src/nvim/os/env.c

```diff
@@ -395,9 +395,16 @@ void os_get_hostname(char *hostname, size_t size)
 
 /// To get the "real" home directory:
 ///   - get value of $HOME
+/// For Windows:
```

Comment:
Rewrite the function description to use an ordered list to disambiguate each step.